astro/src/content/docs/lifecycle/register-users/advanced-registration-forms.mdx

```diff
@@ -27,7 +28,7 @@ Advanced registration forms let you build powerful, multi-step, custom registrat
 
 <img src="/img/docs/lifecycle/register-users/built-out-advanced-form.png" alt="Example of built out advanced registration form." width="1200" />
 
-You might be interested in this feature if you use the FusionAuth themed login pages for your application, and the default self service registration form doesn't meet your needs. Whether you want to break a form up into multiple steps for a better user experience, gather user consents, or have the user provide app specific data, advanced registration forms can help.
+You might be interested in this feature if you use the FusionAuth themed login pages for your application, and the default self service registration form doesn't meet your needs. Whether you want to break a form up into multiple steps for a better user experience, verify identities before user creation, gather user consents, or have the user provide app specific data, advanced registration forms can help.
```

Comment:
I chose not to say "Starting in 1.62.0" for the pre-verified bits in this file but no strong view here.